### PR TITLE
WRO-6381: Fix security vulnerabilities

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
For preventing malware issues, we need to update node modules that fix severity vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update dependencies to the latest versions.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Update the lockfile version to v2

### Links
[//]: # (Related issues, references)
WRO-6381

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
Please rebase your branch.

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
...
-    "@enact/docs-utils": "^0.3.6",
-    "eslint-config-enact-proxy": "^1.0.4",
-    "lerna": "^4.0.0"
+    "@enact/docs-utils": "0.3.6",
+    "eslint-config-enact-proxy": "^1.0.5",
+    "lerna": "^5.1.8"
   }
-}
\ No newline at end of file
+}
```
Comment (MikyungKim) [inline package.json:30]: Let's not making this change.

Comment (hong6316) [reply package.json]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**